CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 
 # PayPal iOS SDK Release Notes
 
+## unreleased
+* PaymentButtons
+	* Change deprecated colors `black`, `silver`, `blue`, and `darkBlue` colors to `white`.
```

Comment:
This PR, since targeting `next-major-version` (V2) should have the breaking change for removing these cases from the enum entirely, not deprecating them

```suggestion
## unreleased
* Breaking Changes
  * PaymentButtons
    * Remove deprecated colors `black`, `silver`, `blue`, and `darkBlue`
```

Reply:
It would probably be easiest to do the deprecation changes on `main` first, update `next-major-version` with the latest `main`, and then remove from `next-major-version`.